packages/flutter/lib/src/cassowary/param.dart

```diff
@@ -6,45 +6,66 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'term.dart';
 
+/// Represents a variable inside the layout solver.
 class Variable {
   static int _total = 0;
 
+  /// Create a new variable with the given constant value.
   Variable(this.value) : _tick = _total++;
 
   final int _tick;
 
+  /// The current value of the variable.
```

Comment:
elaborate on when the value changes, whether it can be set, how it affects the solver